realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -375,7 +375,7 @@ private void injectObjectContext() {
             }
             final Row row = proxyState.getRow$realm();
             if (value == null) {
-                // Table#nullifyLink() does not support default value. Just use Row.
+                // Table#nullifyLink() does not support default value. Just using Row.
```

Comment:
This comment is generated by the annotation processor.
I think that some changes in generator code made by other PR was not applied to those test resouces.